var/spack/repos/builtin/packages/py-cython/package.py

```diff
@@ -15,6 +15,12 @@ class PyCython(PythonPackage):
 
     license("Apache-2.0")
 
+    version("3.0.11", sha256="7146dd2af8682b4ca61331851e6aebce9fe5158e75300343f80c07ca80b1faff")
```

Comment:
There are now two version directives for the same version.  This one needs to be removed.